src/routes/project/[projectId]/settings/slr/+page.svelte

```diff
@@ -2,4 +2,7 @@
     let { data } = $props();
 </script>
 
-<h4>Project {data.projectId} Settings - SLR</h4>
+<svelte:head>
+    <title>SLR • {data.project.name}</title>
```

Comment:
I compared several styles, the separators websites use are `•`, `|` and `/`. I'm fine with `|`.
I like the format `Title | ParentTitle | ParentParentTitle | ... | SnowballR`.

Reply:
The part with title | parent title ... is imo helpful, but do we need to add SnowballR at the end? If it would be website discoverable in the web, ok it makes sense, but in our use case it seems a bit too much, how do you think about it?

Reply:
I looked at it again and agree, I thought we agreed on that but now I remember that we came to that conclusion.

Reply:
Ok (I only mentioned, that other websites sometimes do that and probably to be found more often or so)